dill/tests/_globals_dummy.py

```diff
@@ -0,0 +1,10 @@
+# This file is used by test_shared_globals in test_functions.py
```

Comment:
I think it should be possible to use or modify a simple test like `test_functors` so that file is used as opposed to using an additional file like this one.